src/field.cpp

```diff
@@ -1661,6 +1669,30 @@ bool map::process_fields_in_submap( submap *const current_submap,
                         make_rubble( p );
                         break;
 
+                    case fd_fungicidal_gas:
+                        dirty_transparency_cache = true;
+                        //spread the chemical agent as fast as air would
+                        spread_gas( cur, p, curtype, 60, 40 );
+                        //check the terrain and replace it accordingly to simulate the fungus dieing off
+                        auto ter = map_tile.get_ter_t();
+                        auto frn = map_tile.get_furn_t();
+                        if(  ) {
```

Comment:
Wow, I feel silly. That`s pretty much what I wrote in the very beginning. Where exactly is that flag defined? Feels like I searched everywhere for it.

But thank you.